homework_sample/students.js

```diff
@@ -0,0 +1,46 @@
+console.log("files linked!");
+
+var rawStudentData = "andrew-garver, anila-alexander, ben-sayeg, bryant-novas, dasha-murauyova, david-neuhaus, devatha-kesarkodigae, domenica-lalima, dor-rubin, fernanda-correa, gab-lubliner, gordon-dugan, james-kim, jeremy-bell, john-mcgillion, junho-rhee, karlyce-edwards, kelly-stevens, kelly-devine, mark-paulson, matt-sullivan, max-peterson, michael-dennis, orin-xie, teo-arana, bobby-king, ryan-burke, sam-hu"
+
+var studentstringtoarray = function (string) {
```

Comment:
We want you to practice using camel case, so name it `studentStringToArray` rather than `studentstringtoarray`